metadata_service/proxy/atlas_proxy.py

```diff
@@ -116,22 +142,68 @@ def delete_owner(self, *, table_uri: str, owner: str) -> None:
         pass
 
     def add_owner(self, *, table_uri: str, owner: str) -> None:
-        pass
+        """
+        It simply replaces the owner field in atlas with the new string.
+        FixMe (Verdan): Implement multiple data owners and
```

Comment:
@verdan Is this limitation of Atlas or specific to ING?

Reply:
The owner field type in Atlas is a string by default. I'll have a chat my Bolke and other team members to come up with a better solution.